tasks/data.js

```diff
@@ -82,12 +111,12 @@ async function insertBatch(collection, page, context, task) {
 			? recordsResponse.data.data
 			: recordsResponse.data.data.map((item) => {
 					for (const systemRelation of systemRelationsForCollection) {
-						if (systemRelation.one_collection === "directus_users") {
-							item[systemRelation.many_field] =
-								context.userMap[item[systemRelation.many_field]];
-						} else if (systemRelation.one_collection === "directus_files") {
-							item[systemRelation.many_field] =
-								context.fileMap[item[systemRelation.many_field]];
+						if (systemRelation?.meta?.one_collection === "directus_users") {
+							item[systemRelation?.meta?.many_field] =
```

Comment:
We could rely on `systemRelation.related_collection` / `systemRelation.field` here as well, though it shouldn't matter